test/verify/check-journal

```diff
@@ -136,7 +136,7 @@ ExecStart=/bin/sh -c '/usr/bin/seq 123; sleep 10'
 Description=Slowly log 10 identical lines
 
 [Service]
-ExecStart=/bin/sh -c 'for s in $(seq 10); do echo SLOW; sleep 0.1; done; sleep 10'
+ExecStart=/bin/sh -c 'sleep 5; for s in $(seq 10); do echo SLOW; sleep 0.1; done; sleep 10'
```

Comment:
Ugh, that feels like a systemd journal bug. I don't really like sleeps, but it's harmless enough for now. Thanks!

Reply:
It does a bit but it is super tight race, so almost not possible to create any reasonable reproducer except: run in loop until it hopefully fails.